src/components/events/iwd-26/CredentialCard/CredentialCard.css

```
@@ -0,0 +1,323 @@
+.pc-card * {
+  display: grid;
+  grid-area: 1 / -1;
+  transform: translate3d(0, 0, 0.1px);
+  pointer-events: none;
+}
+
+.pc-inside {
+  inset: 1px;
+  position: absolute;
+  background-image: var(--inner-gradient);
+  background-color: rgba(0, 0, 0, 0.9);
```

Stylelint (17.3.0)

[error] Unexpected duplicate "background-color" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)